{"created": 1745229054.7965157, "duration": 0.8471770286560059, "exitcode": 1, "root": "C:\\Users\\kjeon\\OneDrive\\Desktop\\QA\\Bay", "environment": {}, "summary": {"failed": 1, "total": 1, "collected": 1}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "tests/test_Bay_order_status_delivery.py", "type": "Module"}]}, {"nodeid": "tests/test_Bay_order_status_delivery.py", "outcome": "passed", "result": [{"nodeid": "tests/test_Bay_order_status_delivery.py::test_order_receive_from_delivery", "type": "Function", "lineno": 28}]}], "tests": [{"nodeid": "tests/test_Bay_order_status_delivery.py::test_order_receive_from_delivery", "lineno": 28, "outcome": "failed", "keywords": ["test_order_receive_from_delivery", "test_Bay_order_status_delivery.py", "tests", "Bay", ""], "setup": {"duration": 0.7318226000061259, "outcome": "passed"}, "call": {"duration": 0.0007631999906152487, "outcome": "failed", "crash": {"path": "C:\\Users\\kjeon\\OneDrive\\Desktop\\QA\\Bay\\tests\\test_Bay_order_status_delivery.py", "lineno": 39, "message": "ValueError: \ubc1c\uc8fc \uc9c4\ud589 \uc0c1\ud0dc\uc778 \uc81c\ud488\uc774 \uc5c6\ub2e4"}, "traceback": [{"path": "tests\\test_Bay_order_status_delivery.py", "lineno": 39, "message": "ValueError"}], "stdout": "\u274c Error in test_order_receive_from_delivery: \ubc1c\uc8fc \uc9c4\ud589 \uc0c1\ud0dc\uc778 \uc81c\ud488\uc774 \uc5c6\ub2e4\n", "longrepr": "page = <Page url='about:blank'>\n\n    def test_order_receive_from_delivery(page: Page):\n        try:\n            # product.json\uc5d0\uc11c delivery_status\uac00 3\uc778 \uc81c\ud488\ub4e4 \ucc3e\uae30\n            with open('product_name.json', 'r', encoding='utf-8') as f:\n                products = json.load(f)\n    \n            # delivery_status\uac00 3\uc778 \uc81c\ud488\ub4e4\ub9cc \ud544\ud130\ub9c1\n            eligible_products = [product for product in products if product.get('delivery_status') == 3]\n    \n            if not eligible_products:\n>               raise ValueError(\"\ubc1c\uc8fc \uc9c4\ud589 \uc0c1\ud0dc\uc778 \uc81c\ud488\uc774 \uc5c6\ub2e4\")\nE               ValueError: \ubc1c\uc8fc \uc9c4\ud589 \uc0c1\ud0dc\uc778 \uc81c\ud488\uc774 \uc5c6\ub2e4\n\ntests\\test_Bay_order_status_delivery.py:39: ValueError"}, "teardown": {"duration": 0.06949600006919354, "outcome": "passed"}}]}